.github/workflows/bulk_repo_update.yml

```diff
@@ -0,0 +1,140 @@
+name: Bulk Repo Update
+
+# This workflow is used to run a script on a list of repos and create PRs for the changes introduced by script.
+# Note that this workflow won't work on repos where active branch isn't master.
```

Comment:
Is this still true?  What is the reason for it?

Reply:
Apparently Microsoft is not interested in providing a feature which will make a lot of lives easier. See the ongoing demand for this feature since the start of workflow_dispatch feature https://github.com/orgs/community/discussions/8774. 

On the other hand, since we are separating the workflows for openedx & edx org, one way I can think of is to remove the org parameter and make the openedx/edx as default in respective workflows which will enable us an additional parameter to pass on the branch parameter.

Reply:
Looking through the workflow parameters again, we had this fixed but missed removing the comment. I'll update the comment from the workflow since it now works on the repositories with default branches other than master as well.